src/components/transaction-item.tsx

```diff
@@ -33,6 +33,8 @@ const getRelativeTimestamp = (tx: Transaction | MempoolTransaction) => {
   const date =
     typeof (tx as any).burn_block_time !== 'undefined' && (tx as any).burn_block_time !== -1
       ? toRelativeTime((tx as any).burn_block_time * 1000)
+      : (tx as any).burn_block_time === -1
+      ? 'Need parent burn block time...' // MICROBLOCK TODO: Replace with real data.
```

Comment:
Maybe that's what you meant in your comment, but I think you could use 'parent_burn_block_time' here

Reply:
Yep, I was just waiting for the new deployed version.